modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -0,0 +1,21 @@
+= Connected Apps for Developers and End Users
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Developers
+
+Developers who have websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option.
```

Comment:
Connected Apps does a lot more than Log In with Anypoint. It allows users of the platform to delegate authorization to third-party developer to act on their behalf within the platform

Reply:
```suggestion
Developers who have websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option. Adding this client enables users of the website or web app to allow the third-party developer to act on their behalf.
```